thunder/dynamo/utils.py

```diff
@@ -80,7 +80,10 @@ class SubgraphInfo:
 
     Attributes:
         original_graph_module: The original graph module.
-        split_graph_module: The graph module for the split subgraph.
+        original_split_graph_module: The original split graph module before any transformations are applied.
+            Specifically, before the :func:`checkpoint_converter` replaces the Torch operators with Thunder symbols,
+            and before any submodules are compiled by Thunder.
+        split_graph_module: The graph module for the split subgraph. It contains the compiled thunder/inductor modules.
```

Comment:
I think we should update the docstring for `submodule_to_compiled_functions` to mention that it maps submodules from `original_split_graph_module` to the compiled functions